views/014_job_history_status.sql

```diff
@@ -0,0 +1,60 @@
+CREATE
+OR REPLACE VIEW job_history_latest_status AS
+SELECT
+  jh.*
+FROM
+  job_history jh
+  JOIN (
+    SELECT
+      resource_id,
+      MAX(created_at) AS max_created_at
```

Comment:
We also need status, errors, duration, etc.

Reply:
It's already like that.  All the columns from `job_history` is fetched. You're looking at the intermediate table I'm using to get the latest history per resource.

```
+--------------------------------------+----------------------------+---------------+-------------+------------------+----------+-----------------+-----------------+--------------------------------------+----------+----------------------------+----------------------------+----------------------------+
| id                                   | name                       | success_count | error_count | details          | hostname | duration_millis | resource_type   | resource_id                          | status   | time_start                 | time_end                   | created_at                 |
|--------------------------------------+----------------------------+---------------+-------------+------------------+----------+-----------------+-----------------+--------------------------------------+----------+----------------------------+----------------------------+----------------------------|
| 018704a4-772d-ae27-c1ad-8cb487249faa | SystemTemplateSync         | 1             | 0           | {"errors": null} | Arch     | 14              | system_template | 01870403-e772-44ee-98fd-0f330d651e4b | FINISHED | 2023-03-21 20:33:20.013208 | 2023-03-21 20:33:20.027398 | 2023-03-21 14:48:20.013447 |
| 018704a4-772d-833f-1c90-359e4908963d | ComponentStatusSummarySync | 2             | 0           | {"errors": null} | Arch     | 19              |                 |                                      | FINISHED | 2023-03-21 20:33:20.012519 | 2023-03-21 20:33:20.032311 | 2023-03-21 14:48:20.012833 |
+--------------------------------------+----------------------------+---------------+-------------+------------------+----------+-----------------+-----------------+--------------------------------------+----------+----------------------------+----------------------------+----------------------------+
```